README.md

```diff
@@ -1384,6 +1384,7 @@ _Related: [Wikis](#wikis)_
 - [mojopaste](https://metacpan.org/dist/App-mojopaste) - Perl based pastebin. ([Demo](https://p.thorsen.pm/), [Source Code](https://github.com/jhthorsen/app-mojopaste)) `Artistic-2.0` `Perl`
 - [MokinToken](https://github.com/nexus-uw/mokintoken) - Clientside encrypted pastebin using tweetnacl. `Unlicense` `PHP`
 - [NoteHub](https://github.com/chmllr/NoteHub) - Free and Hassle-free Pastebin for Markdown Pages. Simple, clean, password provided, generated-short link. `MIT` `Nodejs`
+- [paaster](https://paaster.io) - Paaster is a secure by default end to end encrypted pastebin built with the objective of simplicity. ([Source Code](https://github.com/WardPearce/paaster)) `GNU` `Python`
```

Comment:
- License should be `GPL-3.0`
- `end to end` -> `end-to-end`

> The Language tag is the main server-side requirement for the software

- https://github.com/WardPearce/paaster#production -> Docker is the only supported installation method -> please change the language tag to `Docker`

Reply:
Should be resolved in latest commit